buildSrc/src/main/kotlin/native-targets-conventions.gradle.kts

```diff
@@ -45,6 +45,7 @@ kotlin {
         watchosDeviceArm64()
 
         // Deprecated, but not removed
+        @Suppress("DEPRECATION")
```

Comment:
I am not sure this target support is required and maybe we just could remove it?

Reply:
Yes, it is required for one of our customers